timeinterval/timeinterval.go

```diff
@@ -165,6 +172,31 @@ var monthsInv = map[int]string{
 	12: "december",
 }
 
+// UnmarshalYAML implements the Unmarshaller interface for Timezone.
+func (tz *TimeZone) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var str string
+	if err := unmarshal(&str); err != nil {
+		return err
+	}
+
+	loc, err := time.LoadLocation(str)
+	if err != nil {
+		if runtime.GOOS == "windows" && str != "Local" && str != "UTC" {
+			return fmt.Errorf("unable to load time location. Timezones other than 'Local' and 'UTC' may not be supported on Windows without using a custom timezone file: %w", err)
+		}
+		return err
```

Comment:
I'm wondering if we could make the error message more targeted.

```suggestion
		if runtime.GOOS == "windows" {
			if zoneinfo := os.GetEnv("ZONEINFO"); zoneinfo != "" {
				return fmt.Errorf("%w (ZONEINFO=%q)", err, zoneinfo)
			}
			return fmt.Errorf("%w (on Windows platforms, you may have to pass the time zone database using the ZONEINFO environment variable, see https://pkg.go.dev/time#LoadLocation for details)", err)
		}
		return err
```

Reply:
Thanks @simonpasquier. 

Yeah I think that's a good idea. I've also been thinking more, and I'm wondering if we just go the extra distance and embed the timezone database on Windows platforms? Obviously it's not ideal because we can have stale timezone information between releases of AM, but I think that's probably better than nothing. What do you think?

Reply:
I would defer it for a later PR if people feel that it's really needed. As you said, we're at risk of providing stale information.